### DISCUSSION TITLE
Achievements not showing up

### DISCUSSION THREAD

**Discussion by asudox** [community/community#103038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The achievements do not show up on my profile.
I tried going to the achievements tab link manually but I got a 404. The "show achievements on my profile" button is on. I even tried turning it on or off, but it does not seem to work. Help?

**Comment by spenserblack** [comment]
Just starting with the obvious: are you sure you *have* any achievements? Opening pull requests that have been merged, co-authoring commits, answering questions in discussions, or anything like that? Not sure, but some of these might only be counted after a certain period, so IDK if old PRs or other things will apply.

**Reply by asudox** [reply]
When I disable the show achievements button or don't show private contributions, github informs me about how I achieved new achievements, and instructs me to enable the show achievements/private contributions setting to show them.

**Comment by ghostinhershell** [comment]
Hi @Asudox, 

Thanks so much for reaching out in our community. When you enable the ability to display achievements on your profile, are you still unable to see the Achievements that you've earned on your profile?

**Reply by asudox** [reply]
Correct.

**Comment by yeadwastaken** [comment]
ye @Asudox,

have you tried activating "Include private contributions on my profile" under the settings in the Posts & Activity section?

**Reply by asudox** [reply]
Yep.
